client/scss/components/_new-site-notice.scss

```diff
@@ -0,0 +1,58 @@
+$tweakpoints: (
+  'notification-break': 1080px
+);
+
+$mega-w-size: 400px;
+
+.new-site-notice-wrap {
```

Comment:
Would be nice to have e.g. `new-site-notice` and `new-site-notice__inner` with `//* @define new-site-notice` at the top to make sure it stays BEM-ed up in future.